src/WSRequestHandler_Sources.cpp

```diff
@@ -315,6 +315,97 @@ RpcResponse WSRequestHandler::SetVolume(const RpcRequest& request)
 	return request.success();
 }
 
+/**
+* Changes whether an audio track is active for a source.
+*
+* @param {String} `source` Source name.
+* @param {int} `track` Audio tracks 1-6.
+* @param {boolean} `active` Whether audio track is active or not.
+*
+* @api requests
+* @name SetTracks
+* @category sources
+* @since 4.0.0
+*/
+RpcResponse WSRequestHandler::SetAudioTracks(const RpcRequest& request)
+ {
+	if (!request.hasField("source") || !request.hasField("track") || !request.hasField("active")) {
+		return request.failed("missing request parameters");
+	}
+
+	QString sourceName = obs_data_get_string(request.parameters(), "source");
+	bool active = obs_data_get_bool(request.parameters(), "active");
+	int track = obs_data_get_int(request.parameters(), "track")-1;
+
+	if (sourceName.isEmpty()) {
+		return request.failed("invalid request parameters");
+	}
+
+	OBSSourceAutoRelease source = obs_get_source_by_name(sourceName.toUtf8());
+	if (!source) {
+		return request.failed("specified source doesn't exist");
+	}
+
+	uint32_t mixers = obs_source_get_audio_mixers(source);
+	uint32_t new_mixers = mixers;
+
+	if (active)
```

Comment:
Could the audio mixers be pushed out of range by calling the same request twice?